### PR TITLE
Adds Leveraged ETF Strategy 1025

### PR DESCRIPTION
**Pull request by shilewenuw**

### CODE REVIEW

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
+<h3>Inputs/Outputs</h3>
+
+<p>
+    Before we build our Neural Network Architecture, we need to understand the inputs and outputs to our model.
+    The input to the model will be the OHLC+Volume data for t-14 to t time steps (past 15 time steps). The output is a
+    direction (Up, Down, Stationary) of the movement of the average close of the t+1 to t+5 time steps (5 future timestamps).
+    The movement is is considered stationary if the abs(% change 5-step average close) < .01%. These three directions will
```
Comment (DerekMelchin) [inline 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:14]: Typo: "is is"

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+<div class="section-example-container">
+<pre class="python">
+    inputs = Input(shape=(15, 5))
+</pre>
+</div>
+
+
+<p>
+    WWe then feed this Layer into our Convolutional Layer, where we extract features, which will serve as
```
[the remark this answers is not in the document]
Comment (DerekMelchin) [reply 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:60]: This paragraph is a duplicate of the one above.

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+<div class="section-example-container">
+<pre class="python">
+    feature_extraction = Conv1D(30, 4, activation='relu')(inputs)
+</pre>
+</div>
+
+
+<p>
+    We then split this layer into three
```
Comment (DerekMelchin) [inline 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:76]: Duplicate paragraph again.

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+</div>
+
+<p>
+    <em>input_vars</em> define the variables we want to use to make our predictions. The class <em>Direction</em>
+    defines a few integers that we will label our data with (labels are needed for classification problems). The reason
+    we use integers instead of strings is because Keras, like most ML libraries, only work with numerical data. Moving on,
+    <em>rolling_avg_window_size</em> is the number of time steps used for the calculate the average of future closing prices,
+    described earlier in
+    <b>Inputs/Outputs</b> (t+1 to t+5 is 5 time steps, and our value accordingly is set to 5).
+    The constant stationary_threshold defines the threshold for a change to be considered
```
Comment (DerekMelchin) [inline 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:160]: Should stationary_threshold be wrapped in "em" tags too?

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+<div class="section-example-container">
+<pre class="python">
+    df['close_avg'] = df['close'].rolling(window=rolling_avg_window_size).mean().shift(shift)
+    df['close_avg_change_pct'] = (df['close_avg'] - df['close']) / df['close']
+</pre>
+</div>
+
+<p>
+    The rolling mean should be self explanatory for those familiar with pandas (if not, I hope by now readers realize
+    this is definitely not a beginner-friendly resource).
```
Comment (DerekMelchin) [inline 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:179]: "definitely not a beginner-friendly resource" may sound a little harsh to some. Maybe favor "a more advanced tutorial"?

File: 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+    Here, <em>.shift(shift)</em> aligns the five time step rolling average 'close_avg' column to the end of the last
```
Comment (DerekMelchin) [inline 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:180]: We were using "em" tags before but now we wrap close_avg in single-quotes. It's probably better to keep things consistent.

Comment (shilewenuw) [reply 04 Strategy Library/1024 Forecasting Stock Prices using a Temporal CNN/03 Method.html:180]: Thanks!, I've implemented all the fixes

**Pull request closed without merging**